packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -100,6 +100,26 @@ function getFrameworkTests(opts: {
 				},
 				flags: ["--style", "sass"],
 			},
+			react: {
+				promptHandlers: [
+					{
+						matcher: /Select a variant:/,
+						input: [keys.enter],
+					},
+				],
+				unsupportedOSs: ["win32"],
+				unsupportedPms: ["yarn"],
+				testCommitMessage: true,
+				verifyDeploy: {
+					route: "/",
+					expectedText: "Vite + React",
+				},
+				verifyPreview: {
+					route: "/",
+					previewArgs: ["--host=127.0.0.1"],
```

Comment:
This is necessary because by default `vite preview` is only listening on `localhost`, which doesn't appear to include `127.0.0.1` (at least on some OSes).